custom/rch/models.py

```diff
@@ -0,0 +1,212 @@
+    Child_Aadhaar_No = models.BigIntegerField(null=True)
+    Child_RCH_ID_No = models.BigIntegerField(null=True)
+    DPTBooster1_Antibiotics_Given = models.CharField(null=True, max_length=255)
+    DPTBooster1_Breastfeeding = models.CharField(null=True, max_length=255)
+    DPTBooster1_Complentary_Feeding = models.CharField(null=True, max_length=255)
+    DPTBooster1_Diarrhoea = models.CharField(null=True, max_length=255)
+    DPTBooster1_ORS_Given = models.CharField(null=True, max_length=255)
+    DPTBooster1_Pneumonia = models.CharField(null=True, max_length=255)
+    Delete_mother = models.CharField(null=True, max_length=255)
+    Fully_Immunized = models.CharField(null=True, max_length=255)
+    Gender = models.CharField(null=True, max_length=255)
+    MDDS_VillageID = models.IntegerField(null=True)
+    Measles1_Antibiotics_Given = models.CharField(null=True, max_length=255)
+    Measles1_Breastfeeding = models.CharField(null=True, max_length=255)
+    Measles1_Complentary_Feeding = models.CharField(null=True, max_length=255)
+    Measles1_Diarrhoea = models.CharField(null=True, max_length=255)
+    Measles1_ORS_Given = models.CharField(null=True, max_length=255)
+    Measles1_Pneumonia = models.CharField(null=True, max_length=255)
+    Mobile_Relates_To = models.CharField(null=True, max_length=255)
+    Mother_Case_no = models.SmallIntegerField(null=True)
+    Mother_RCH_ID_NO = models.BigIntegerField(null=True)
+    Name_Child = models.CharField(null=True, max_length=255)
+    Name_Father = models.CharField(null=True, max_length=255)
+    Name_Mother = models.CharField(null=True, max_length=255)
+    Received_AllVaccines = models.CharField(null=True, max_length=255)
+    SubCentre_ID = models.SmallIntegerField(null=True)
+    VILLAGE_Name = models.CharField(null=True, max_length=255)
+    Weight = models.FloatField(null=True)
+
+    @classmethod
+    def update_beneficiaries(cls):
+        date_str = str(datetime.date.fromordinal(datetime.date.today().toordinal()-1))
+        for state_id in STATE_DISTRICT_MAPPING:
+            for district_id in STATE_DISTRICT_MAPPING[state_id]:
+                records = fetch_child_beneficiaries_records(date_str, date_str, state_id, district_id)
+                for record in records:
+                    rch_child = cls()
+                    for prop in record:
+                        rch_child.__setattr__(prop.keys()[0], prop.values()[0])
+                    rch_child.save()
+
+
+class AreaMapping(models.Model):
+    stcode = models.IntegerField(null=False)
+    stname = models.CharField(max_length=255, null=False)
+    dtcode = models.IntegerField(null=False)
+    dtname = models.CharField(max_length=255, null=False)
+    pjcode = models.IntegerField(null=False)
+    pjname = models.CharField(max_length=255, null=False)
+    awcid = models.BigIntegerField(null=False)
+    awcname = models.CharField(max_length=255, null=False)
+    villcode = models.IntegerField(null=False)
```

Comment:
Its just to keep it same as the fields received from the API as shared [above](https://github.com/dimagi/commcare-hq/pull/14554#discussion-diff-96421945R32)